domains/wisit.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "wisit moondet",
+  "repo": "https://github.com/Wisitt/wisit",
+  "owner": {
+    "username": "Wisitt",
+    "email": "wisitmoondet@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://wisit.vercel.app/"
```

CNAME can't contain protocol or slashes
```suggestion
    "CNAME": "wisit.vercel.app"
```